SparkPost/2web2ui
# SA-1330: Updated active filter styling

### What Changed
 -  Hibana active filters updated for Report builder
- Report builder changed to functional component

### How To Test
 - Go to report builder (/report/summary for now)
 - Add filters
 - Check that styling matches
 - Test different filter types
 - Check that removing filters still works
